crates/relayer-cli/src/commands/clear.rs

```diff
@@ -135,6 +135,8 @@ impl Runnable for ClearPacketsCmd {
         let opts = LinkParameters {
             src_port_id: self.port_id.clone(),
             src_channel_id: self.channel_id.clone(),
+            max_memo_size: config.mode.packets.ics20_max_memo_size.get_bytes(),
+            max_receiver_size: config.mode.packets.ics20_max_receiver_size.get_bytes(),
```

Comment:
```suggestion
            max_memo_size: config.mode.packets.ics20_max_memo_size.get_bytes() as usize,
            max_receiver_size: config.mode.packets.ics20_max_receiver_size.get_bytes() as usize,
```

Reply:
Fixed in 0fd448cd8e91d9a825357093c9694cc2e817e6af